packages/transition-backend/src/tasks/dataImport/importPlacesFromGeojson.ts

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2025, Polytechnique Montreal and contributors
+ *
+ * This file is licensed under the MIT License.
+ * License text available at https://opensource.org/licenses/MIT
+ */
+import GenericDataImportTask from 'chaire-lib-common/lib/tasks/dataImport/genericDataImportTask';
+import importGeojsonPlaces from '../../services/importers/PlacesDataSourceImporter';
+
+export default class ImportPlacesFromGeojson extends GenericDataImportTask {
+    constructor(fileManager: any) {
+        super(fileManager);
+    }
+
+    /**
+     * IN: A file {@link GenericDataImportTask#POINT_OF_INTEREST_FILE} and
+     * {@link GenericDataImportTask#RESIDENTIAL_ENTRANCES_FILE} containing the
+     * residences and poi points
+     *
+     * OUT: The tr_places table is populated with the data from the geojson files
+     *
+     * @param dataSourceDirectory The directory containing the data sources
+     */
+    protected async doRun(dataSourceDirectory: string): Promise<void> {
+        // Import from the point of interest file
+        const absoluteDsDir = this._importDir + dataSourceDirectory + '/';
+        const POIgeojsonFilePath = absoluteDsDir + GenericDataImportTask.POINT_OF_INTEREST_FILE;
+        const dataSourceName = `${dataSourceDirectory} Places`;
+
+        await importGeojsonPlaces(POIgeojsonFilePath, dataSourceName);
```

Comment:
Do those 2 imports need to be sequential ? Or can we do them in "parallel"?  We could "await all" at the end of the function (Maybe with a flag in the task to select selection if we prefer, for debug purpose)

Reply:
For now, they need to be sequential. The import function asks to overwrite the data source and if they run in parallel, there will be 2 questions at the same time and you don't know what you're answering to. If we eventually split the data source creation/overwrite from the import, then the import part can be done in parallel. But as currently, this task is the only one doing the import, the split can wait for another use case.